resources/views/listing/partials/filter/category.blade.php

```diff
@@ -1,10 +1,47 @@
+<ais-hierarchical-menu
+    :attributes="[
+        'category_lvl1',
+        'category_lvl2',
+        'category_lvl3',
+        {{-- TODO: This should be dynamic up to the max depth --}}
+    ]"
+    {{-- root-path="Men > Tops" --}}
+    show-more
+>
+    <template v-slot="{ items, refine, createURL }">
+        {{-- See the HierarchicalMenuList.vue --}}
+        <recursion :data="items" v-slot="{ data, component }">
+            <ul>
+                <li v-for="item in data" :key="item.value">
+                    <a
+                        :href="createURL(item.value)"
+                        :class="{ 'font-bold': item.isRefined }"
+                        v-on:click.exact.left.prevent="refine(item.value)"
+                    >
+                        @{{ item.label }}
+                        (@{{ item.count }})
+                    </a>
+                    {{--
+                    TODO: Double check this, the data updates,
+                    but the component doesn't. Tried to
+                    make everything working recursive
+                    plus keep it renderless.
```

Comment:
This usually is solved by adding a `:key` that will change depending on the data so the element is forcibly rerendered: https://vuejs.org/api/built-in-special-attributes#key